samples/steps/http-post.md

```diff
@@ -24,9 +24,9 @@ post_step:
 
 * `args`
     * `query`
-        * *..desired query values*
+        * *..desired query values* - Scripts can be used for query values
```

Comment:
@ewertub please explain "Scripts can be used for query values"

Reply:
Here is an example:
`
query:
      some_val: ${incoming.query.parameters}
      another_val: ${assignedValue}
`